### PR TITLE
Constraints for non-leptonic class-I decays

### PR DESCRIPTION
**Pull request by smeiser**
- Added constraints on B_(s) -> D_(s)^(*) form factors relevant for class-I non-leptonic decays
- Added constraints on B_(s) -> D_(s)^(*) P branching ratios

### CODE REVIEW

File: eos/constraints/B-to-V-form-factors.yaml [hunk 1]
```diff
@@ -757,6 +757,35 @@ B_s->D_s^(*)::FormFactors[parametric,LCSRLattice]@BGJvD:2019A:
     - [ 0.019937564067748218, 0.9959435000075986, -2.0693925361777366, 0.005209708684882278, -1.6091113066762444, 0.05763619287102934, -13.511549611076651, 228.6192228860033, 0.2563621121382167, -12.316356789452744, 64.61794723355086, 0.129911934046514, 1.959575188793536, -9.556234622420847, -1.2863345122728966, 71.33151044983076, 0.2392745530285085, -8.773719473087793, 58.60772386003895, 0.06490802872581866, 1.4911006984176782, 101.73479766353098, -10.486133354309, 156.35363152316438, 21.151448554579826, -3504.1737951670407, 60536.72271509368 ]
 # }}}
 
+###################################
+# B_(s) -> D_(s)^(*) form factors #
+###################################
+# {{{
+B_(s)->D_(s)^(*)::FormFactors[f_0(Mpi2),f_0(MK2),A_0(Mpi2),A_0(MK2)]@MvDV:2024A:
```
Comment (dvandyk) [inline eos/constraints/B-to-V-form-factors.yaml:764]: @smeiser I would label these differently. The suffix indicates our ongoing project, but you derived these values from BGJvD:2019A, correct? Please use the latter reference name, indicating attribution to the other authors of that work.

**Review by dvandyk — CHANGES_REQUESTED** [review]

File: eos/constraints/B-to-V-form-factors.yaml (hunk 1 above)
Comment (smeiser) [reply eos/constraints/B-to-V-form-factors.yaml:764]: Yes, you are right. but I don't find this article in the EOS reference list, let me add it real quick.

File: eos/references.yaml [hunk 2]
```diff
@@ -289,6 +289,13 @@ BFvD:2016A:
     id: oai:arXiv.org:1608.07127
   inspire-id: Boer:2016iez
   title: QCD Factorization Theorem for $B \to \pi \pi \ell \nu$ Decays at Large Dipion Masses
+BGJvD:2019A:
+  authors: Bordone, M. and Gubernari, N. and van Dyk, D. and Jung, M.
+  eprint:
+    archive: arXiv
+    id: oai:arXiv.org:hep-ph/1912.09335
```
Comment (dvandyk) [inline eos/references.yaml:296]: This is incorrect I think. Older entries are ``hep-ph/YYMMABC``. Newer ones are simply ``YYMM.ABCDE``.

**Review by dvandyk — CHANGES_REQUESTED** [review]

File: eos/references.yaml (hunk 2 above)
Comment (smeiser) [reply eos/references.yaml:296]: Ah sure, changed it.

**Review by dvandyk — APPROVED** [review]

**Pull request merged**